### PR TITLE
feat(footers): merge footers into epic - INNO-898

### PR DESCRIPTION
**Pull request by raduchiriac**
# PR description

Please drop a few lines about the PR: what it does, how to test it, etc.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

* [ ] `package.json` is up-to-date and `@ec-europa/ecl-base` is part of the dependencies
* [ ] I have checked the dependencies
* [ ] I have given the fractal status “ready” to my component
* [ ] I have declared `@define mycomponent` in the SCSS file
* [ ] I have specified `margin: 0;` on the CSS component
* [ ] I have provided tests
* [ ] I follow the naming guidelines
* [ ] the component supports composition
* [ ] there are no hardcoded strings (all content come from the context)
* [ ] I have filled the README.md file (at least a few lines)
* [ ] My component is listed in the root README
* [ ] My PR has the right label(s)

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 122539efeb185a5587a9dbade99db6cdd04325fd

https://deploy-preview-594--europa-component-library.netlify.com

File: src/flavors/eu/components/ecl-footers-eu/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Footers
+# EU Footers
```
Comment (kalinchernev) [inline src/flavors/eu/components/ecl-footers-eu/README.md]: When sharing preview with design team, a request for updated specification would possibly make the documentation actually useful.

File: src/flavors/eu/components/ecl-footers-eu/ecl-footers-eu.config.js [hunk 2]
```diff
@@ -10,8 +10,9 @@ module.exports = {
         wrapper_class: 'ecl-footer__site-identity',
         menus: [
           {
+            extra_classes: 'ecl-col-sm',
             title: {
-              classes: 'ecl-h4',
+              classes: 'ecl-h4 ecl-footer__link--side-identity',
```
Comment (kalinchernev) [inline src/flavors/eu/components/ecl-footers-eu/ecl-footers-eu.config.js]: I'd suggest "side" to be "site" to be move consistent with `.ecl-footer__site-identity`, if there is no other specific reasons to be "side" of course.

File: src/flavors/eu/components/ecl-footers-eu/package.json [hunk 3]
```diff
@@ -1,16 +1,13 @@
...
+  "version": "0.0.1",
+  "description": "EU Footer",
+  "main": "ecl-footers-eu.scss",
+  "style": "ecl-footers-eu.scss",
   "dependencies": {
-    "@ec-europa/ecl-base-eu": "^0.0.1",
-    "@ec-europa/ecl-grid-eu": "^0.0.1",
-    "@ec-europa/ecl-links-eu": "^0.0.1",
-    "@ec-europa/ecl-typography-headings-eu": "^0.0.1"
+    "@ec-europa/ecl-base": "^0.10.0"
```
Comment (kalinchernev) [inline src/flavors/eu/components/ecl-footers-eu/package.json]: This one needs to be `@ec-europa/ecl-base-eu"` I guess.

File: src/flavors/eu/components/ecl-footers-eu/package.json [hunk 4]
```diff
@@ -1,16 +1,13 @@
...
-  "version": "0.0.1",
-  "description": "ECL Footers",
-  "main": "ecl-footers.scss",
-  "style": "ecl-footers.scss",
   "author": "European Commission",
   "license": "EUPL-1.1",
+  "version": "0.0.1",
+  "description": "EU Footer",
+  "main": "ecl-footers-eu.scss",
+  "style": "ecl-footers-eu.scss",
```
Comment (kalinchernev) [inline src/flavors/eu/components/ecl-footers-eu/package.json]: Not sure whether it worths keeping this `ecl-footers.scss` as the other one. Maybe we a discussion for naming of these files will be needed together with the overall naming topic around flavors/families.

**Pull request merged**